src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,218 @@
 
-<DemoCard
-    repository={'side-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-full-config'}
-    react
-/>
+-   fewer than seven top-level pages
+-   primary pages with labels that require complex icons or may be too long to wrap effectivly
+-   limited space but require a nested navigation hierarchy
+
+<React.Fragment>
+    <ImageGrid
+        images={[CollapsibleImg]}
+        caption={[
+            'Figure 1 displays the normal collapsed state of the menu.',
+            'Figure 2 displays the open state of the menu.',
+        ]}
+        regularWidth
+    />
+    <ImageGrid
+        images={[PersistentImg, PermanentImg]}
+        caption={
+            'When the user hovers on the Navigation Rail, the side navigation will expand temporarily, revealing the text details of each navigation item; when the user moves their cursor away, the whole side navigation panel collapses.'
+        }
+    />
+</React.Fragment>
+
+## Temporary
+
+A temporary drawer is invisible until it is opened — typically by pressing a button in the [App Bar](/patterns/appbar). This variation of the drawer appears on top of the main navigation, typically using a dark overlay to block interaction with the main content. Use this variation if you have:
+
+-   a mobile application
+-   extreme space limitations
+
+<React.Fragment>
+    <ImageGrid images={[NavigationTemporaryImg]} regularWidth />
+    <Divider />
+</React.Fragment>
+
+# Tab Navigation
+
+Tab navigation uses a horizontal bar with buttons for each navigation location. These can appear at the top or bottom of your application. Tabs can be used in conjunction with drawer navigation as a way to navigate between sub-pages (e.g., you may have several sub-pages on your main settings page).
+
+<ImageGrid images={[NavigationTabsImg]} regularWidth />
+
+<React.Fragment>
+    <MaterialDesignDescription title={'Tabs'} url={'https://material.io/components/tabs'} />
+    <MaterialDesignDescription title={'Bottom Navigation'} url={'https://material.io/components/bottom-navigation'} />
+</React.Fragment>
 
-In this implementation, the navigation panel slides out from the side, partially obscuring the main content of the application. It is often implemented with a dark overlay that blocks the main application and directs the user's focus to the navigation panel that was opened.
+# Behaviors
 
-## Icon Navigation Bar / Responsive Navigation
+## Nested Navigation
+
+When using drawer navigation, you can have multiple levels of navigation hierarchy. These appear as expandable panels in the drawer. When using a hierarchical navigation in the drawer, you should try to limit yourself to now more than two levels.
+
+<ImageGrid images={[NestedItemImg, NestedItemStandardsImg]} regularWidth />
```

Comment:
![image](https://user-images.githubusercontent.com/8997218/94948180-e81c8000-04ac-11eb-87a5-2779293f07a6.png)

The documentations here are inaccurate ("BG White 200" "Inactive" etc)

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/8997218/94962785-03938500-04c5-11eb-934b-58bb0b636ec1.png)
This _active item_ is body 1 but semibold, it rests on a 5% blue500 background

![image](https://user-images.githubusercontent.com/8997218/94962810-0e4e1a00-04c5-11eb-9cac-dc9eb425976f.png)
This is body 1 / subtitle 1, they are the same

![image](https://user-images.githubusercontent.com/8997218/94962831-1efe9000-04c5-11eb-9fb0-423ef2e257f2.png)
This is body 1 SB on top of a body 2